lib/executor.js

```diff
@@ -168,23 +168,24 @@ function setupModels(app, instructions) {
 }
 
 function defineModels(app, instructions) {
+  var registry = app.registry || app.loopback;
   instructions.models.forEach(function(data) {
     var name = data.name;
     var model;
 
     if (!data.definition) {
-      model = app.loopback.getModel(name);
+      model = registry.getModel(name);
       if (!model) {
         throw new Error('Cannot configure unknown model ' + name);
       }
       debug('Configuring existing model %s', name);
     } else if (isBuiltinLoopBackModel(app, data)) {
```

Comment:
Maybe we should create a new registry for built-in models that stands on its own (eg. `loopback.internal = new Registry()`